jetty-core/jetty-quic/jetty-quic-common/src/main/java/org/eclipse/jetty/quic/common/QuicStreamEndPoint.java

```diff
@@ -258,7 +258,8 @@ public void fillInterested(Callback callback)
     {
         if (LOG.isDebugEnabled())
             LOG.debug("setting fill interest on {}", this);
-        super.fillInterested(callback);
+        if (!isFillInterested())
```

Comment:
Why is it needed? At least add a comment?

Reply:
Most H3 tests fail without this change, investigating why...